src/main/java/bit/javaoop/Manager.java

```diff
@@ -20,6 +20,8 @@ public void hire(Employee employee) {
         employees.add(employee);
     }
 
+    public abstract boolean canHire(Employee employee);
```

Comment:
Taa to jest to :D Teraz Managerowie muszą zaimplementować `canHire`, bo inaczej program nie ruszy xd